n2y/main.py

```diff
@@ -24,9 +24,13 @@ def main(raw_args, access_token):
     )
     parser.add_argument("config", help="The path to the config file")
     parser.add_argument(
-        "--verbosity", '-v', default='INFO',
+        "--verbosity", '-ver', default='INFO',
         help="Level to set the root logging module to",
     )
+    parser.add_argument(
```

Comment:
I'd try to pull the version number out of setup.py somehow, so we don't have to update it in two places (we'll probably forget if we have to).